### DISCUSSION TITLE
Shmupmame

### DISCUSSION THREAD

**Discussion by Frankless** [OnionUI/Onion#496] [Ideas]
I just stumble on [this](https://shmupmame.wordpress.com/about/) i think it could be a nice addition :)

**Comment by Znidz** [comment]
Dude, yes please!     
But I think this would be dependent on this particular custom version of MAME having a libreto core and it seems abandoned.

**Comment by MrFaust** [comment]
I've been building my own but based on the FBAlpha2012 core because I've noticed on the mame2003plus core the audio pitch is wrong and speed is around 55-58fps. I tried the FB Neo core and its running at 38-50fps and audio is slow of course. Once I get it completed I will package up the files and submit. I'll also include a .dat file so you can build a rom set for it. Right now I'm trying to get the aspect ratio issues addressed. Some cores stretch the game to fit the full screen. Some squeeze the game. 
![arcadeshmups](https://user-images.githubusercontent.com/68164545/174341088-e362634a-e128-4aed-8348-0f6278810a98.png)
![arcadeshmups_sel](https://user-images.githubusercontent.com/68164545/174341089-c9708556-3c82-4656-ba81-ee608a91ec06.png)
.
